src/main/java/controllers/SHHController.java

```diff
@@ -0,0 +1,146 @@
+package controllers;
+
+import com.jfoenix.controls.JFXListView;
+import com.jfoenix.controls.JFXTextField;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import models.HouseRoomsModel;
+import models.RoomModel;
+
+import java.io.IOException;
+import java.util.Set;
+
+public class SHHController {
+    private static Main mainController;
+    @FXML private Label errorLabel;
+    @FXML private JFXTextField temperatureInput;
+    @FXML private JFXListView zoneRoomList;
+    private ObservableList<String> observableZoneList= FXCollections.observableArrayList();
+    HouseRoomsModel houseRoomsModel = HouseRoomsModel.getInstance();
+    RoomModel [] roomModels = houseRoomsModel.getAllRoomsArray();
+
+    /**
+     * keep an instance of the mainController
+     * @param main
+     */
+    public void setMainController(Main main){
+        mainController=main;
+    }
+
+    /**
+     * update the zone room view after finish adding zone and room
+     * @param observableZoneRoomList
+     */
+    public void updateListView(ObservableList<String> observableZoneRoomList){
+        zoneRoomList.setItems(observableZoneRoomList);
+    }
+
+    /**
+     * open the new set up zone room window
+     * @param event
+     */
+    public void handleZoneRoom(ActionEvent event) {
+        // check permission before open this
+        try {
+            mainController.setZoneRoomWindow(this);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * set the morning temperature to the selected zone and room
+     * each room have an instance of the Temperature so change the new value with the existing one
+     * @param event
+     */
+    public void setMorningTemperature(ActionEvent event) {
+        temperatureInput.clear();
+        if(!checkSelectZone() || !checkValue()){
+            return ;
+        }
+        String selectZone= (String) zoneRoomList.getSelectionModel().getSelectedItem();
+        double inputT = Double.parseDouble(temperatureInput.getText());
```

Comment:
fix typo or provide better naming for "inputT".

Reply:
it not type it is inputTemperature but it would too long